modules/ols-configuring-openai-using-projects.adoc

```diff
@@ -0,0 +1,34 @@
+:_mod-docs-content-type: PROCEDURE
+[id="ols-configuring-openai-using-projects_{context}"]
+= Configuring OpenAI using projects
+
+This procedure explains how to configure {openai} using projects. Projects provide the ability to organize your work. 
+
+If you will not use projects or if {openai} is not the LLM provider that you use then skip this task.
+
+.Prerequisites
+
+* You have an {openai} account and valid log in credentials.
+
+.Procedure
+
+. In the OpenAI portal, go to the *Settings* and create a project. 
+
+. Give the project a name. For example, OpenShift Lightspeed Project.
+
+. Cick *API keys*.
+
+. Click *Create new secret key*.
+
+. Select *Service account*.
+
+. Enter a unique ID for the Service account. For example, such as openshift-lightspeed.
```

Comment:
, for example, `openshift-lightspeed`.

Reply:
I deleted the "such as" and left this as two sentences. There is a good chance this topic will not be used and will be deleted.